frida_mode/GNUmakefile

```diff
@@ -116,7 +125,8 @@ ifndef OS
  $(error "Operating system unsupported")
 endif
 
-GUM_DEVKIT_VERSION=15.1.22
+
+GUM_DEVKIT_VERSION=15.1.25
```

Comment:
Why the version bump? No reason not to use the latest, but are there any new features/fixes in FRIDA which you need? Think there is a newer version still? I assume Ole has been busy as he's a bit behind on the release notes.

Reply:
Happy with whatever version you want to go with. Just wondered if there was any problems with 15.1.27 which means that 15.1.25 was preferabe.